### PR TITLE
Feature/org install fixes

### PR DESCRIPTION
**Pull request by Lance-Simeon**

### CODE REVIEW

File: SimeonInstaller.ps1 [hunk 1]
```diff
@@ -1847,6 +1847,8 @@ CRLFOption=CRLFAlways
             [string]$Organization,
             # Disconnects the Dev Ops organization from Azure AD (Beware when using this option as you can lose access to the org)
             [switch]$DisconnectDevOpsOrganizationFromAzureAd,
+            # Exits the installer after the org and tenant are created. Used primarily to prepare orgs in order to send to Microsoft for unblocking free tier
+            [switch]$ExitAfterDevOpsProjectCreation,
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:1851]: why not just have a separate function to create the org and project?

File: SimeonInstaller.ps1 [hunk 2]
```diff
@@ -2088,12 +2089,35 @@ CRLFOption=CRLFAlways
...
+            $collectionBuildService = ($users |? displayName -like "Project Collection Build Service (*").descriptor
+            Invoke-WithRetry { Invoke-RestMethod -Header $authenicationHeader -Uri "https://vssps.dev.azure.com/$Organization/_apis/graph/memberships/$collectionBuildService/$contributorsgroupDescriptor`?api-version=6.1-preview.1" -Method Put } | Out-Null
+
             # Repositories > Permissions > Contributors > allow Create repository
             Set-AzureDevOpsAccessControlEntry -Organization $Organization -ProjectId $projectId -SubjectGroupPrincipalName "[$Project]\Contributors" -PermissionNumber 256 -PermissionDescription "Create Repository"
             Set-AzureDevOpsAccessControlEntry -Organization $Organization -ProjectId $projectId -SubjectGroupPrincipalName "[$Project]\Contributors" -PermissionNumber 8 -PermissionDescription "Force Push"
             Set-AzureDevOpsAccessControlEntry -Organization $Organization -ProjectId $projectId -SubjectGroupPrincipalName "[$Project]\Contributors" -PermissionNumber 16384 -PermissionDescription "Administer build permissions"
         }
 
+        # Invite users to tenants contributor group
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:2103]: as admin or as contributor?

Comment (Lance-Simeon) [reply SimeonInstaller.ps1:2103]: As a member of the contributors group, so users can approve pipelines

Comment (admin-simeon) [reply SimeonInstaller.ps1:2103]: so why is the parameter called InviteToOrgAsAdmin?

**Pull request merged**